### DISCUSSION TITLE
New Contributor Introductions!

### DISCUSSION THREAD

**Discussion by mikhael28** [mikhael28/paretOS#17] [Announcements]
Let's use this thread as a place to highlight who we are, what interests us about the project and any ideas we have moving forward as to what you want to work on!

Let's start with myself: my name is Michael, but my friends and colleagues call me Misha. I'm an open-source developer based in Seattle, who is passionate about technical education & continuous improvement. I started this project as a business venture, but I changed it to an open-source project after I was inspired by watching a Ted talk by Linus Torvalds. In it, he mentioned how it was great working on something alone, but once you build something substantial enough - you want to show it off, and you want to work with other people on it to get their feedback, ideas and such! I hope that the ParetOS can be for high-level, browser based operating systems what Linux was for low-level languages. Ambitious, yes, unrealistic, probably, but who cares - let's create something extraordinary! I am also starting the open-source.js Meetup in Seattle in September 2021, and would love to answer any questions contributors have.

**Comment by dylvaz** [comment]
Hello all 👋! My name is Dylan. I'm a self-taught developer with a love for M/PERN and AWS (recently certified 😄). I stumbled across ParetOS on the issues page and not only did I like the tech stack, but also the educational value this can bring to new developers. I'm looking forward to contributing more.

**Reply by mikhael28** [reply]
I just merged your pull-request, meaning you have the 'honor' of being the first contributor to this project!! Thank you, it gives me more energy to contribute further. Any ideas for what you might want to work on next? Have you had a chance to create an account? Right now, I am going through components/code I haven't touched in a while and making tickets in preparation for introducing this project to some local developer communities to find more passionate developers interested in the project. So, lots of documentation writing in the coming days (for example, the 'Paywall Puzzle' for the educational curriculum - why is that there?)

**Reply by dylvaz** [reply]
I have created an account! I would like to work on the refactoring of components from class to hooks. I'll take a peek at some of the issues we have open as well. I'll be sure to open some tickets if I come up with suggestions.

**Reply by mikhael28** [reply]
That sounds great - the way you refactored the Login component was perfect. A lot of the issues are pretty standard, more or less simple fixes. I've created an issue label called 'Ownership' - it's taking ownership of a complete feature that we want implemented, from end to end (with my assistance, of course). Completion of a feature like that qualifies you to be a part of the 'core team', or whatever. Basically, having 'contributor' access to the repository. I will definitely look forward to seeing any tickets you write up - I am slowly working my way through the repository, writing up the @TODO items as tickets and fleshing them out a bit more, while expanding on the 'vision' of the product, especially the 'ownership' tickets. Even if you aren't planning on working on those, it is good to take a look at them just to understand what the OS's vision (from my perspective, at least) is. Always keeping the 80/20 principle in mind.

**Comment by dl90** [comment]
Hi I'm Don and I'm here to practice reading code and fixing bugs on large React projects.
I want to thank you for taking the time to put this project up, encouraging and allowing new devs to practice and learn.

About me, I've graduated from a 2 year diploma for full stack web development back in May and I'm working towards landing a junior position.

**Reply by mikhael28** [reply]
That sounds great Dan, happy to have you aboard! Thank you for your first PR that you sent in, I am going to be reviewing your second in a few hours!

**Comment by shubhankarb180** [comment]
Hi everyone 🤘 !! I am Shubhankar, a front end developer from India. I have been procrastinating since long about Open Source contribution but finally decided to do it and find a project. I was lucky to find a issue and have submitted a PR. I still found contributing lot fun and learn some new things while solving the issue.  I maybe a beginner but I would love to learn more about OSS. So hear read and solve all the issues which I can have my bucket 🙂

**Reply by mikhael28** [reply]
Hey Shubhankar, it's great to have you here! I just left a few comments on your PR that you submitted, almost done! This thread is a pretty good way for us to communicate until we set up a more effective system. I am right now hesitant to use a Slack/Discord/real-time chat because I think that adds more stress to everyone's lives. I am interested to see how far we can go just using GitHub discussions and issues. Also, congrats - you are the third open-source dev to contribute to the project! Look forward to working together.

**Reply by shubhankarb180** [reply]
Cheers @mikhael28

**Comment by Chris-isCoding** [comment]
console.log(`Hello world`);

I discovered joy of programming in January when I was learning how to declare a variable in JavaScript. 
Currently I am taking baby-steps learning React.

Unfortunately lately super busy with work (no-tech role) but I am hoping to contribute whenever I can and learn from others.

**Reply by mikhael28** [reply]
Glad to have you Chris! All the way from Poland :D Let me know if there is a particular concept/API/React thing you want to work on and practice.

**Comment by wesley3295** [comment]
Hello Everyone,
I am Wesley, a Jr  Full Stack Developer recently graduated from Flatiron School, in April. I am however located in Arizona. I am currently looking for a remote or in state job opportunity based around my skill-set. I have knowledge in Ruby, Rails, SQL, JavaScript, and React/Redux. 

This was my first project I contributed to. I'm really looking forward to learning more and bettering my skill-set. Thanks for allowing me to contribute to this project @mikhael28.

**Reply by mikhael28** [reply]
Hey Wesley, great to have you here! Looking forward to see what you can accomplish here with us :)

**Comment by nikhilsdabhade** [comment]
Hi everyone,
i'm Nikhil, a react developer from india. i started learning react and made some projects and i always wanted to contribute to open source and then i found this project and really liked the tech stack.
i'm a beginner in this field and would love to learn a lot from you guys!

**Reply by mikhael28** [reply]
Great to meet you! Check out the Issues, and comment on which one you want to work on/what questions you might have about it.

**Comment by Youspeakcubic** [comment]
Hey guys, My name is Anthony, I'm an aspiring front end developer. I am currently in college for computer programming, and am taking bootcamp courses to try to land a junior position. I would say I'm fairly proficient in html, css, and javascript, and love solving issues in these languages. I would really love to better learn react as i am very much a beginner and start moving towards fullstack maybe through PHP or java. Looking forward to working with so many talented developers on such a cool project :).  Have a wonderful day and may your code be error free.

**Reply by mikhael28** [reply]
Thanks Anthony, great working with you so far! Is there anything in React you are looking to learn about in particular?

**Reply by Youspeakcubic** [reply]
I would really like to get more practice working with state, and i'm also wanting to learn hooks as i'm basically clueless as to what those are right now.

**Reply by mikhael28** [reply]
Alright, let me think about a good thing for you to work on.

**Reply by Youspeakcubic** [reply]
Thank you, I appreciate it 😃. I would love to help out anyway i can just let me know if you find anything. I was also looking at the issue on a mobile revamp, i was brainstorming and have a few ideas. I'm a little busy today but ill try to make a wire frame and some mock ups tomorrow and post them on the issue page.

**Reply by mikhael28** [reply]
Yeah, that would be fantastic. Mobile navigation is critical, we need a simple, one page way of getting to most of the application.

**Reply by mikhael28** [reply]
@Youspeakcubic any ideas on that mobile revamp? You ever see Windows Phone 7 UI?

**Comment by Leoactionz** [comment]
Hi everyone 🤓 I am Leo, an autodidact♌️, UX designer🎨 and aspiring developer👨🏻‍💻 from Nigeria. I love learning and implementing new stuff. New to react, and I see I have a lot to learn. An OS where a complete newbie can pick up things to do, fantastic. Need to pick up more of the UI issues as well as functionality, especially those that can fit into my plate.

**Reply by mikhael28** [reply]
We are really glad to have you here! I have heard nothing but great things about Nigeria's tech scene, it's going to be ridiculously huge in a few years. I would also encourage you to look at some of the 'ownership' issues, that are a big bigger in scope - but since you are new to React, picking up smaller pieces to build your confidence, knowledge and muscle memory is probably for the best. Congratulations on your first merged pull-request! What are some of the interesting trends coming out of Nigerian tech in the last few years?

**Reply by Leoactionz** [reply]
I will be happy to touch on the "ownership" issues while I pick up the smaller tasks to boost confidence. For Nigerian tech, payment is doing the magic now with fintech, telemedicine is not fully harnessed and AgroTech is coming up. ttys

**Reply by mikhael28** [reply]
What are some of the popular fintech startups in Lagos?

**Reply by Leoactionz** [reply]
> 
> 
> - [Infibranches Technologies](infibranches.com) 
> - [ReelFruit ](http://reelfruit.com/) 
> - [Edves](https://edves.net/) 
> - [Mono](mono.co) 
> - [Zilla](https://zilla.africa/)

 To mention a few.

**Comment by nexusocean8** [comment]
Hello all :wave: My name Vincent and I'm from Florida. I've been working in tech for a few years and I'm an aspiring fulltime web dev. I'm most familiar with React frameworks and TypeScript. I also hold an AWS cert. I'm new to open source and hoping to work on some exciting projects!

**Reply by mikhael28** [reply]
You from Miami by any chance? My mentor, who got me in the game years ago, just moved there from Trinidad and Tobago. He says it's quite the hotspot in tech right now (mostly Web3/crypto stuff).

What kind of companies have you been looking at right now - you actively interviewing? Have you checked out the list of job boards we are curating in the ParetOS library of context?

**Reply by nexusocean8** [reply]
Miami is a cool place and crypto is an interesting topic, but I'm actually near Tampa, FL. I work remotely as a production support engineer, which is a somewhat vague role, at Olive AI. I plan to stay with company for a while and hopefully grow with them before I consider applying to any outside roles. I'll definitely check out the job boards a bit more though.

**Reply by mikhael28** [reply]
Olive AI - I work at a company called Cedar AI, they sound like sister companies! Glad to hear you like where you work, I love where I'm at too.

**Comment by pskyechology** [comment]
Greetings everyone! I am Asko, a software development student from 🇪🇪 (some of our popular startups are: Bolt, Wise and Veriff), Tallinn, with my primary focus on all things JavaScript. Most recently I've started familiarising myself with next.js and React to try and make some awesome, modern websites. Open source is not new to me, but contributing to open source projects is. I figured it was about time I start giving back to the community that has given me so many awesome projects to use. 😄

**Reply by mikhael28** [reply]
Thank you @asko1 ! My grandmother was born in Estonia, a country very near and dear to my heart! What do you think about the project so far?

**Comment by eddybruv** [comment]
Hello everyone, I'm Edwin (my friends call me Eddy😅), a MERN Stack developer from Cameroon 🇨🇲. I love JS but recently started learning TypeScript. I stumbled upon this project using the issue tracker, took a look at the code and found it interesting so I decided to contribute to this awesome project. Hope I create good friendship/professional bonds while working on this project and write good code too. And most importantly, grow individually and create value for others🙂.

**Reply by mikhael28** [reply]
Cameroon! That is flipping awesome - tell us more about the tech scene there! Who are some of the big companies in the area/hot startups?

**Reply by eddybruv** [reply]
> 

There are a few hot startups around my area but my favorite has to be [Nkwa](https://mynkwa.com/).

**Reply by mikhael28** [reply]
That's a pretty interesting company - kind of like a bank, or interest earning savings account? We are working on developing the Library of Context, an International collection of resources to help people understand the start-up ecosystem in every country! For example, check out Nigeria's collection -  https://www.paret0.com/hubs/nigeria

Would you be interested in being the 'curator' for the Cameroon collection?
